editor/modes/text-editor/style.scss

```diff
@@ -132,3 +132,9 @@
 	max-width: $text-editor-max-width;
 	margin: 0 auto;
 }
+
+@media (max-width: 599px) {
```

Comment:
I think we ought to define this style mobile-first with `margin-top: 56px;` and a media query to reset on larger displays. There are a number of [mixins](https://github.com/WordPress/gutenberg/blob/e869a9dcde45197feb9d1ebc1b45047c1da551a5/editor/assets/stylesheets/_mixins.scss#L1-L35) and [variables](https://github.com/WordPress/gutenberg/blob/e869a9dcde45197feb9d1ebc1b45047c1da551a5/editor/assets/stylesheets/_variables.scss#L111-L116) available for this. I think you'll want:

```scss
margin-top: 56px;

@include break-small {
	margin-top: 0;
}
```

...inside the `.editor-layout__editor` selector.